components/datetime/tests/fixtures/tests/components-partial-matches.json

```diff
@@ -119,5 +119,22 @@
                 "en":  "week 1 of December"
             }
         }
+    },
+    {
+        "description": "Singleton fallback for F (F -> d)",
```

Comment:
Question: Do we have other patterns in the test data that resolve to `F`? It would be nice to exercise more of the pattern matching machinery if possible. I haven't looked at the shape of the CLDR data here.

Suggestion: If we already have this data in the test data it we should have a test for it.

Suggestion (follow-up): If we don't have this data, but it is available in CLDR, then we should import more data to make sure we have test coverage for it.

What are your thoughts here?

Reply:
CLDR doesn't use 'F' at all in its patterns so only singletons (~ #1581) currently work as test vectors. (append item support #586 should improve that but that'd test the append machinery rather than pattern matching).